com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -56,45 +56,65 @@ public enum ScriptableRenderPassInput
         /// </summary>
         AfterRenderingPrePasses = 200,
 
+        /// <summary>
+        /// Executes a <c>ScriptableRenderPass</c> before rendering gbuffer pass.
+        /// </summary>
+        BeforeRenderingGbuffer = 250,
```

Comment:
@phi-lira modifying the enum values break serialized enum values in existing project (ex: UniversalRenderingExamples). Should I squeeze the 4 new enum values between the range 200-250 instead?